sklearn/ensemble/_stacking.py

```diff
@@ -526,6 +532,13 @@ class StackingClassifier(ClassifierMixin, _BaseStacking):
     0.9...
     """
 
+    _parameter_constraints = {
+        **_BaseStacking._parameter_constraints,  # type: ignore
+        "stack_method": [
```

Comment:
I think that we can move this parameter into the base class as well. It will be set for the regressor but will have one of the accepted value.